gcc/config/aarch64/cygming.h

```diff
@@ -318,6 +318,15 @@ do {							\
 
 #define HAVE_64BIT_POINTERS 1
 
+extern void i386_pe_asm_lto_start (void);
+extern void i386_pe_asm_lto_end (void);
```

Comment:
Extern declarations should not be in `cygming.h` as it's causing build issues for Cygwin and MSYS2 targets, they should be in `aarch64-protos.h`. I have an open PR for that https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/pull/1.

Reply:
Yes, it was added temporary. Declarations are in mingw/winnt,h